src/openforms/js/components/admin/form_design/form-creation-form.js

```diff
@@ -1001,6 +1001,9 @@ const FormCreationForm = ({formUuid, formUrl, formHistoryUrl}) => {
     },
     newForm: !formUuid,
   };
+  const adminBaseUrl = `${window.location.origin}/admin`;
+  const adminRequestsLog = `${adminBaseUrl}/log_outgoing_requests/outgoingrequestslog`;
```

Comment:
it's possible the app is hosted on a subpath, so assuming `/admin/` as root is too optimistic.

The most robust way is to pass this as a `data-` attribute in the django template `src/openforms/forms/templates/admin/forms/form/change_form.html`

```django
data-outgoing-requests-url="{% reverse 'admin:...' %}"
```

and then consume that and pass it down in `src/openforms/js/components/admin/index.js`

This way you also avoid hardcoding the URLs entirely :)